app/theme/client/imports/components/sidebar/sidebar-item.css

```diff
@@ -316,6 +316,37 @@
 @media (width <= 400px) {
 	.sidebar-item {
 		padding: 0 0 0 var(--sidebar-small-default-padding);
+		&__message {
+			display: flex;
+			flex: 1;
+			margin: 0 -3px;
+			align-items: center;
+			justify-content: space-between;
+			overflow: visible !important;
+			&-top {
+				overflow: visible !important;
+				width: 100%;
+			} 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4892745)